docs/README.md

```diff
@@ -45,7 +45,9 @@ The subdomain used for the project can be configured via the `modules.local-serv
 
 ## Available Commands
 
-* `composer server start [--xdebug]` - Starts the containers. If the `--xdebug` option is passed the PHP container will have XDebug enabled. To switch off XDebug run this command again without the `--xdebug` option.
+* `composer server start [--xdebug]` - Starts the containers.
+  * If the `--xdebug` option is passed the PHP container will have XDebug enabled. To switch off XDebug run this command again without the `--xdebug` option.
+  * Any files in `content/upload` are synced to the S3 server on start up.
```

Comment:
I don't think we should add this behaviour for `start` by default, at least now. I'd say we should keep it as it's own command.

Reply:
Why so in your view? I don’t disagree necessarily, my thinking was that it’s just one less thing to think about when starting the server.

To be fair it is weird to have docker compose services that immediately exit.

Reply:
I removed this, I think you're right, it's less magical.